process-services/2.4/support/index.md

```diff
@@ -53,6 +53,7 @@ Choose a combination of products to build your own Supported Stack. If anything
 | **Integrations** | |
 | Identity Service 1.2 | For use with LDAP and SAML |
 | Identity Service 1.1 | For use with LDAP and SAML |
+| Keycloak 24.0.3 | Enable `Exclude Issuer from Authentication Response` - see [Configure authentication](https://docs.alfresco.com/process-services/2.4/config/authenticate/#properties){:target="_blank"} for details. |
```

Comment:
1. Update the link format for "internal" docs, so it's verified by our build process.
2. Copy the fixed note to the **Process Services 2.4** entry in `content-services/7.4/support/index.md`. _This completes the ACS part of your updates._
3. Agree with Aritri how to complete the rest of the **APS** part of the docs update.

```suggestion
| Keycloak 24.0.3 | Enable `Exclude Issuer from Authentication Response` - see [Configure authentication]({% link process-services/2.4/config/authenticate.md}#properties) for details. |
```

Reply:
The instructions here are very much more summarized compared to what has been written in the DOCS ticket, I'm not sure it'll be easy for customers to follow through.

cc: @vprovaggi

Reply:
The more detailed instructions will be provided in this "Configure authentication" section.